sunpy/net/dataretriever/sources/stereo.py

```diff
@@ -146,16 +148,16 @@ def _get_url_for_timerange(self, timerange, **kwargs):
             raise ValueError('Earliest date for which PLASTIC data is available is {:%Y-%m-%d}'.format(START_DATE))
         prefix = 'http://stereo-ssc.nascom.nasa.gov/data/beacon/{source}/plastic/'
         prefix_copy = prefix
-        if (source == 'STEREO_A'):
-            prefix += '%Y/%m/ST{char}_LB_PLASTIC_%Y%m%d_'
-            prefix_copy += '%Y/%m/ST{char}_LB_PLA_BROWSE_%Y%m%d_'
-        elif (source == 'STEREO_B'):
-            prefix += '%y/%m/ST{char}_LB_PLASTIC_%Y%m%d_'
-            prefix_copy +='%y/%m/ST{char}_LB_PLA_BROWSE_%Y%m%d_'
+        source_dict = {'STEREO_A':'%Y', 'STEREO_B':'%y'}
+        prefix += source_dict[source]
+        prefix_copy += source_dict[source]
+        prefix += '/%m/ST{char}_LB_PLASTIC_%Y%m%d_'
+        prefix_copy += '/%m/ST{char}_LB_PLA_BROWSE_%Y%m%d_'
```

Comment:
why repeating the same operations over two things when you could do it once and then change it for the end.
Mostly when the only thing it changes is `PLASTIC` to `PLA_BROWSE`
Also, it would be more descriptive if you call prefix, pattern and suffix copies more descriptive.